views/pages/home.ejs

```diff
@@ -25,9 +25,15 @@
     <div class="main">
         <div class="column-articles">
         <% posts.forEach(post => { %>
-            <article>
-                <h2 id=<%= post.id %>><a href="/posts/<%= post.slug %>"> <%= post.title %></a></h2>
+            <article id="article-<%= post.id %>">
+                <div id=<%='post-'+ (Number(post.id)) %> class="article-bottom-padding"></div>
```

Comment:
nu e nevoie să-l forțezi să fie Number, că oricum faci concatenare. Dar îți lipsesc ghilimelele la id.